Jint.Tests/project.json

```diff
@@ -1,19 +1,15 @@
 {
 	"version": "1.0.0-*",
   "dependencies": {
-    "Microsoft.Data.Sqlite": "1.0.0-*",
-    "Dapper.Dnx": "1.43.0-beta2",
-    "xunit.runner.dnx": "2.1.0-beta6-build191",
-    "xunit": "2.1.0"
+    "xunit.runner.dnx": "2.1.0-*",
+    "xunit": "2.1.0-*",
+    "Jint": "2.7.2-*",
```

Comment:
What's the reason to increment this version? I don't really know what the recommendation is when referencing local project which are also on nuget. I would assume `current-*`. I will ask the team.